src/main/java/games/strategy/engine/framework/startup/ui/ClientSetupPanel.java

```diff
@@ -298,16 +298,16 @@ public IChatPanel getChatPanel() {
     if (!isServerHeadless) {
       return new ArrayList<>();
     }
-    final List<Action> rVal = new ArrayList<>();
-    rVal.add(clientModel.getHostBotSetMapClientAction(this));
-    rVal.add(clientModel.getHostBotChangeGameOptionsClientAction(this));
-    rVal.add(clientModel.getHostBotChangeGameToSaveGameClientAction());
-    rVal.add(clientModel.getHostBotChangeToAutosaveClientAction(this, SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE));
-    rVal.add(clientModel.getHostBotChangeToAutosaveClientAction(this, SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE2));
-    rVal.add(clientModel.getHostBotChangeToAutosaveClientAction(this, SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE_ODD));
-    rVal.add(clientModel.getHostBotChangeToAutosaveClientAction(this, SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE_EVEN));
-    rVal.add(clientModel.getHostBotGetGameSaveClientAction(this));
-    return rVal;
+    final List<Action> actions = new ArrayList<>();
+    actions.add(clientModel.getHostBotSetMapClientAction(this));
+    actions.add(clientModel.getHostBotChangeGameOptionsClientAction(this));
+    actions.add(clientModel.getHostBotChangeGameToSaveGameClientAction());
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE));
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE2));
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_ODD));
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_EVEN));
+    actions.add(clientModel.getHostBotGetGameSaveClientAction(this));
+    return actions;
```

Comment:
Using Arrays.asList would remove 2 lines of code

Reply:
I'm going to pass on this one because callers may be dependent on this method returning a mutable collection and `Arrays#asList()` returns a pseudo-immutable collection (unless it is subsequently wrapped by an  `ArrayList`, which kind of defeats the point).

Currently, the only caller of this method does not depend on the mutability of the collection, but this is an interface method that imposes no immutability requirement on the implementation.  Establishing a new interface requirement and updating all implementations and callers is out of scope for this PR. :smile: